src/site/_includes/partials/post.njk

```diff
@@ -54,6 +54,32 @@
       </div>
     {% endif %}
 
+    {% if translation %}
+      {% if translation === true %}
+        <div class="w-banner w-banner--warning w-banner--body">
+          This page is not translated yet.
```

Comment:
This means all articles will initially have a 'this article has not been translated yet banner', right?

Earlier you mentioned that we might end up in a situation where an article hasn't been machine translated yet, but someone submits a translation, and then the machine translation gets submitted shortly thereafter.

What if we only show the gitlocalize link after the article has gotten its initial machine translation? And change the language to something like "We're currently working on an initial translation for this article."

Reply:
I think manual (human) translation is usually better than machine one? Should we deter someone from posting their translation because we can get machine translation later on? (Machine one does not automatically overwrite what is already present, it submits a PR just like a human would do)

Reply:
I agree that a human translation is better. My concern was if someone submitted a pull request that only translated a portion of the article, and then at the same time we receive the machine translation. Which of the two pull requests would we choose to merge?

This might be an edge case though. So I think we can go with the text that you have and if it causes issues we can always tweak it.